SuffolkLITLab/docassemble-MAEvictionMoratorium
# New language re: security screen with image. Fixes #61

Ask if user has permission to share codefendants' names and info publicly, etc. Fixes #61 

Test 1:
- See the correct test on the page.
- Make sure asking about codefendants still works as expected.

I'm making the PR, but it's not my code, so I'll review it and merge it too.
